transport/src/main/java/io/netty/channel/pool/SimpleChannelPool.java

```diff
@@ -101,6 +101,51 @@ protected void initChannel(Channel ch) throws Exception {
         });
     }
 
+    /**
+     * Returns the {@link Bootstrap} this pool will use to open new connections.
+     *
+     * @return the {@link Bootstrap} this pool will use to open new connections
+     *
+     * @since 4.1.7
```

Comment:
I don't mean to be passive-aggressive about this. It was just my best guess at the time of writing; happy to change it as needed.